components/automate-cli/cmd/chef-automate/status.go

```diff
@@ -75,6 +119,41 @@ func getStatus() (*api.StatusResponse, error) {
 	return res, nil
 }
 
+func runStatusSummaryCmdFunc(statusSummaryCmdFlags *StatusSummaryCmdFlags) func(cmd *cobra.Command, args []string) error {
+	return func(cmd *cobra.Command, args []string) error {
+		return executeStatusSummary(cmd, args, statusSummaryCmdFlags)
+	}
+}
+
+func executeStatusSummary(cmd *cobra.Command, args []string, statusSummaryCmdFlags *StatusSummaryCmdFlags) error {
+	if isA2HARBFileExist() {
+		infra, err := getAutomateHAInfraDetails()
+		if err != nil {
+			return err
+		}
+		statusSummary := NewStatusSummary(infra, FeStatus{}, BeStatus{}, 10, time.Second, statusSummaryCmdFlags)
+		sshUtil := statusSummary.(*Summary).getSSHConfig()
+
+		err = statusSummary.Prepare(sshUtil)
+		if err != nil {
+			return err
+		}
+		if isManagedServicesOn() {
+			fe := statusSummary.ShowFEStatus()
+			fmt.Println(fe)
+		} else {
+			fe := statusSummary.ShowFEStatus()
+			fmt.Println(fe)
+			be := statusSummary.ShowBEStatus()
+			fmt.Println(be)
+
+		}
+	} else {
```

Comment:
You can avoid this else by returning inside if and then just returning outside if

Reply:
Done